src/connectors/sort-by-selector/connectSortBySelector.js

```diff
@@ -157,6 +157,30 @@ export default function connectSortBySelector(renderFn, unmountFn) {
         unmountFn();
         return state.setIndex(this.initialIndex);
       },
+
+      getWidgetState(fullState, { state }) {
```

Comment:
According to the change in `getWidgetSearchParameters`:

- `fullState` should be renamed `uiState`
- `state` should be renamed `searchParam`